### ISSUE TITLE
Map not loading for some users

### ISSUE THREAD

**Issue by samuk** [transitionnetwork/TNv3.0#100]
User 1 Chrome
Have tried restart F5 etc.
http://whatsmybrowser.org/b/ZZCO92O
![ambermap](https://cloud.githubusercontent.com/assets/400875/20630838/6a24d2d2-b32b-11e6-999b-19f266b4141d.png)

User 2 Safari
"I tried again today after a complete shutdown and reboot and still no joy in terms of that map appearing.
I am using Safari if that is of any consequence"

**Comment by yvonnestruthers** [comment]
Hi Sam,

None of us can replicate the issue here. I do notice that one of the users is using Windows 7 which may be part of the problem but I will ask Tom's thoughts on Monday.

**Comment by samuk** [comment]
I can't replicate it either. It seems to work here:

https://www.browserstack.com/start#os=Windows&os_version=7&browser=Chrome&browser_version=54.0&zoom_to_fit=true&full_screen=true&resolution=responsive-mode&url=https%3A%2F%2Ftransitionnetwork.org%2Ftransition-near-me%2F&speed=1&host_ports=google.com%2C80%2C0

My only thought is perhaps on a slow connection the page times out?

**Comment by arhinostorm** [comment]
Hi all,

Believe I've found and fixed the problem. To replicate it you had to be logged in. When logged in the map wouldn't load, and also on the homepage the carousel wouldn't load... I think these explain the safari issues as well and the homepage not loading. It wasn't anything to do with safari - everything to do with being logged in and the changes I made for SSL.

I've fixed and tested this now. Should be working properly on all browers.
